template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CentralCmTemplateUpdater.java

```diff
@@ -37,6 +37,9 @@
     @Inject
     private CmTemplateComponentConfigProcessor cmTemplateComponentConfigProcessor;
 
+    @Inject
+    private CmTemplateConfigInjectors cmTemplateConfigInjectors;
```

Comment:
Not a big deal but this could have been better named `CmTemplateConfigInjectorProcessor` by analogy of `CmTemplateComponentConfigProcessor`.